debugger/source.ml

```diff
@@ -55,10 +58,6 @@ let source_of_module pos mdle =
           try find_in_path_normalized path (innermost_module ^ ext)
           with Not_found -> loop exts
     in loop source_extensions
-  else if Filename.is_relative fname then
-    find_in_path_rel path fname
-  else if Sys.file_exists fname then fname
```

Comment:
Looking just at the diff shown by github, I don't understand what happens in the new version on this case (`fname` exists and is an absolute path).

Reply:
Ask for a few more line above the patch :). This was done twice. The function used to be 
```
if Sys.file_exists fname then fname
else if [...]
else if Sys.file_exists fname then fname
else [...]
```
So I simply removed this second occurrence